src/App.jsx

```diff
@@ -1,10 +1,14 @@
 /* eslint-disable import/no-named-as-default */
 import React from 'react';
 import './App.scss';
+import './auth/passwordreset/CSS/reset.scss';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import HeaderSection from './commons/components/HeaderSection';
 import FooterSection from './commons/components/FooterSection';
 import Landing from './Landing/containers/Landing';
+import ResetPasswordContainer from './auth/passwordreset/containers/ResetContainer';
+import 'react-toastify/dist/ReactToastify.css';
+import {ToastContainer} from 'react-toastify';
```

Comment:
@HabibSentongo I guess you get rid of toastify since the PO suggested to go by displaying the error/status messages on the same page with in the form containers

Reply:
Both these issues have been resolved, thank you @alexxsanya